chain/consensus/filcns/upgrades.go

```diff
@@ -2753,10 +2753,19 @@ func PreUpgradeActorsV16(ctx context.Context, sm *stmgr.StateManager, cache stmg
 	if err != nil {
 		return xerrors.Errorf("error getting lookback ts for premigration: %w", err)
 	}
+	timeout := os.Getenv("LOTUS_MIGRATE_PROGRESS_LOG_SECONDS")
+	timeoutDuration, err := time.ParseDuration(timeout + "s")
+	if err != nil {
+		return xerrors.Errorf("error parsing LOTUS_MIGRATE_PROGRESS_LOG_SECONDS: %w", err)
+	}
+
+	if timeoutDuration == 0 {
+		timeoutDuration = time.Second * 2
+	}
```

Comment:
you're not allowing for the case of _not_ providing the env var, so it'll error unless you set it because you can't parse `"s"`

```suggestion
	timeoutDuration := time.Second * 2
	timeout := os.Getenv("LOTUS_MIGRATE_PROGRESS_LOG_SECONDS")
	if timeout != "" {
		parsed, err := time.ParseDuration(timeout + "s")
		if err != nil {
			return cid.Undef, xerrors.Errorf("error parsing LOTUS_MIGRATE_PROGRESS_LOG_SECONDS: %w", err)
		}
		timeoutDuration = parsed
	}
```